css/signup.css

```diff
@@ -0,0 +1,58 @@
+@import url("../css/utility.css");
+@import url("../css/components/navbar.css");
+@import url("../css/components/footer.css");
+
+/* Hero */
+#hero {
+  background-image: url("../assets/homepage/homepage-banner.svg");
+  background-repeat: no-repeat;
+  background-position: right;
+  object-fit: contain;
+  min-height: 70rem;
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  justify-content: center;
+  position: relative;
+}
+
+.overlay {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.1);
+}
+
+.input-wrapper {
+  position: absolute;
+  left: 16%;
+  top: 5%;
+  background-color: var(--white);
+  z-index: 5;
+}
+
+.tnc-wrapper {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 0.5rem;
```

Comment:
suggestion  - use px if the unit is less than 1rem.

Reply:
Done as suggested.